manifests/ResponsivelyApp/ResponsivelyApp/0.1.6.yaml

```diff
@@ -0,0 +1,17 @@
+Id: ResponsivelyApp.ResponsivelyApp
+Version: 0.1.6
+Name: ResponsivelyApp
+Publisher: ResponsivelyApp
+License: © 2020 ResponsivelyApp, Inc. All rights reserved.
+AppMoniker: ResponsivelyApp
+Tags: responsively,app,viewpoint
+Description: A modified browser built using Electron that helps in responsive web development.
+Homepage: https://manojvivek.github.io/responsively-app/
+Installers:
+  - Arch: x64
+    Url: https://github.com/manojVivek/responsively-app/releases/download/v0.1.6/ResponsivelyApp-Setup-0.1.6.exe
```

Comment:
This URL 404's, according to the [releases](https://github.com/manojVivek/responsively-app/releases) page of ResponsivelyApp, version 0.1.6 hasn't been released yet.

Did you mean to add version 0.1.5? (https://github.com/manojVivek/responsively-app/releases/download/v0.1.5/ResponsivelyApp-Setup-0.1.5.exe)

Reply:
Actually it was 0.1.6, now the author changed to 0.1.5. I have updated the correct version.